tests/TestCase/Mailer/fakeMailAttachment.php

```diff
@@ -0,0 +1,8 @@
+<?php
```

Comment:
`tests/TestCase` should not contain test data. Also creating a new file isn't required, just use any php file from `tests/test_app/config` for e.g.

Reply:
what about
* [tests/TestCase/Http/server_mocks.php](https://github.com/cakephp/cakephp/blob/5.x/tests/TestCase/Http/server_mocks.php)
* [tests/TestCase/Validation/stubs.php](https://github.com/cakephp/cakephp/blob/5.x/tests/TestCase/Validation/stubs.php)

then?